comp/api/api/apiimpl/api_test.go

```diff
@@ -92,10 +72,17 @@ func getComponentDependencies(t *testing.T) testdeps {
 		fetchonlyimpl.MockModule(),
 		fx.Supply(context.Background()),
 		taggerimpl.MockModule(),
+		fx.Provide(func(mock tagger.Mock) tagger.Component {
+			return mock
+		}),
```

Comment:
Ideally, these lines should be moved inside `taggerimpl.MockModule()` as `tagger.Mock` should also provide an implementation for tagger.Component